kore/src/Kore/ASTVerifier/AttributesVerifier.hs

```diff
@@ -106,3 +131,46 @@ verifyNoHookAttribute attributes = do
             return ()
         Just _ ->
             koreFail "Unexpected 'hook' attribute"
+
+
+verifyAxiomAttributes
+    :: forall error a
+    .  MonadError (Error VerifyError) error
+    => IndexedModule Verified.Pattern Attribute.Symbol a
+    -> Attribute.Axiom SymbolOrAlias
+    -> error (Attribute.Axiom Internal.Symbol.Symbol)
+verifyAxiomAttributes indexedModule axiom = do
+    let overload = axiom Lens.^. field @"overload"
+    case getOverload overload of
+            Nothing -> do
+                let newOverload = Overload Nothing
+                return (axiom & field @"overload" Lens..~ newOverload)
+            Just (symbolOrAlias1, symbolOrAlias2) -> do
+                symbol1 <- toSymbol symbolOrAlias1
+                symbol2 <- toSymbol symbolOrAlias2
+                let newOverload = Overload $ Just (symbol1, symbol2)
+                return (axiom & field @"overload" Lens..~ newOverload)
+  where
+    toSymbol :: SymbolOrAlias -> error Internal.Symbol.Symbol
+    toSymbol symbolOrAlias = do
+        (symbolAttributes, decl) <-
+            resolveSymbol indexedModule symbolConstructor
+        symbolSorts <-
+            symbolOrAliasSorts (symbolOrAliasParams symbolOrAlias) decl
+        let symbol =
+                Internal.Symbol.Symbol
+                    { symbolConstructor
+                    , symbolParams
+                    , symbolAttributes
+                    , symbolSorts
+                    }
+        return symbol
+      where
+        symbolConstructor = symbolOrAliasConstructor symbolOrAlias
+        symbolParams = symbolOrAliasParams symbolOrAlias
+
+verifySymbolAttributes :: a
```

Comment:
Let's give this a signature like `verifyAxiomAttributes`, something like

```.hs
verifySymbolAttributes
    :: forall error a
    .  MonadError (Error VerifyError) error
    => IndexedModule Verified.Pattern Attribute.Symbol a
    -> Attribute.Symbol
    -> error Attribute.Symbol
verifySymbolAttributes _ attrs = return attrs
```

and also for `verifySortAttributes`.